pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -165,12 +171,14 @@ class CollectionViewSet(
     ViewSet for Collections.
     """
 
-    authentication_classes = []
-    permission_classes = []
```

Comment:
This will make the views accessible for admin only, right? Unless we inject an access policy stating otherwise.

Reply:
That is my understanding.

Reply:
That's correct

Reply:
Don't we need to provide a default permissive access policy then to keep pulp_ansible upstream happy until we implement RBAC there?

Reply:
@mdellweg @bmbouter  I'm more than happy to add these. Just let me know if you want them.

Reply:
I believe we need them in order not to change current behaviour. And i think they should be like allow everything. Also galaxy overrides them anyway, right?
I just want a second opinion before i make it a requirement.

Reply:
After some matrix convo, I believe we're all in agreement these lines should be removed.